src/lib.rs

```diff
@@ -56,15 +57,16 @@ struct LazVlr {
 #[pymethods]
 impl LazVlr {
     #[new]
-    fn new(record_data: &PyAny) -> PyResult<Self> {
+    fn new<'py>(record_data: &Bound<'py, PyAny>) -> PyResult<Self> {
         let vlr_data = as_bytes(record_data)?;
         let vlr = laz::LazVlr::read_from(vlr_data).map_err(into_py_err)?;
         Ok(LazVlr { vlr })
     }
 
     #[classmethod]
-    fn new_for_compression(
-        _cls: &PyType,
+    #[pyo3(signature = (point_format_id, num_extra_bytes, use_variable_size_chunks=false))]
```

Comment:
Assuming that `use_variable_size_chunks=False` is the default, given the `use_variable_size_chunks.unwrap_or(false)` call at L78 below.

Reply:
If it's just false/true, we could make it a `bool` instead of `Option`?

Reply:
yes

Reply:
Ok, changed to `bool` at commit 56753f78f054bacb12724f7ef2102084a59c18fc.